### ISSUE TITLE
2.2M04 Browser holds session data and does not ask for new password

### ISSUE THREAD

**Issue by ikwattro** [neo4j/neo4j#4012]
This issue does not annoy me as I know it, but people may be annoyed by it.

How to reproduce : 

* Download and install neo4j 2.1.7
* Launch the server
* Access the browser at http://localhost:7474
* Close the neo4j browser tab
* Download and install neo4j 2.2M04
* Launch the server
* Open new browser tap at http://localhost:7474
* Neo4j does not ask you for new password, a browser refresh is needed to get the good screen

![screen shot 2015-02-14 at 01 35 05](https://cloud.githubusercontent.com/assets/1222009/6197709/98b587aa-b3ea-11e4-82c9-ce78b66ec492.png)

OS X Yosemite - Chrome Browser

**Comment by cybersam** [comment]
I saw a similar issue, except that the login screen was seriously schizophrenic. It was simultaneously asking me to sign in AND telling me (twice) that I was already signed in. I was not able to sign in or make queries until I refreshed the screen.

Also, even after the browser page was working fine, I had a similar problem when I went to the webadmin page. It kept telling me to log in on the browser page, but of course I was already logged in. Again, refreshing the webadmin page fixed this issue.
